README.md

```diff
@@ -1,14 +1,18 @@
 # MoveIt! Tutorials
 
-This repo is automatically built by the ROS build farm and its output is hosted here: http://docs.ros.org/indigo/api/moveit_tutorials/html/
+This repo is automatically built by the ROS build farm and its output is hosted here: http://docs.ros.org/kinetic/api/moveit_tutorials/html/
```

Comment:
Does the buildfarm generate documentation for packages that are not released for this distribution yet?
The tutorials depend on pr2_arm_kinematics that is not released for kinetic.